pkgs/applications/networking/browsers/brave/default.nix

```diff
@@ -127,6 +128,9 @@ in stdenv.mkDerivation rec {
         # Replace xdg-settings and xdg-mime
         ln -sf ${xdg_utils}/bin/xdg-settings $out/opt/brave.com/brave/xdg-settings
         ln -sf ${xdg_utils}/bin/xdg-mime $out/opt/brave.com/brave/xdg-mime
+
+        wrapProgram "$out/bin/brave" \
+          --suffix XDG_DATA_DIRS : ${gtk3}/share/gsettings-schemas/${gtk3.name}/
```

Comment:
We have a `wrapGAppsHook` that is doing exactly that and other fixes related to gtk: https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/instant-messengers/rambox/default.nix#L31 

this also requires: https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/instant-messengers/rambox/default.nix#L33

Reply:
I tried that first, but unfortunately it doesn't do the trick. None of the other chromium-based browsers seem to use it, as well.

Reply:
Adding `glib` is mandatory otherwise `wrapGAppsHook` will not add gsettings-schemas.

Reply:
I probably should've taken a look at the source of wrapGAppsHook. Thanks for your help, it works and should be done now! :D

Reply:
It also took me a bit to figure that out. That should really go to a gtk documentation chapter in nixpkgs/doc